spidermon/contrib/scrapy/monitors/monitors.py

```diff
@@ -633,7 +633,7 @@ def get_threshold(self):
         crawler = self.data.crawler
         prev_item_scraped_count = self.stats.get("prev_item_scraped_count", 0)
         item_scraped_count = self.stats.get(self.stat_name)
-        crawler.stats.set_value("prev_item_scraped_count", item_scraped_count)
+        crawler.stats.set_value("prev_item_scraped_count", item_scraped_count or 0)
```

Comment:
:lipstick: Wouldn’t adding `0` to the earlier `get()` work? i.e. doing in line 635 the same as in line 634.

Reply:
Yes, I wasn't sure which place was better to apply the fix. I decided to do it in 636 just for the remotely case where you may want to keep the original value of `item_scraped_count` for who knows what reason.